Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Processing/ImageConvert.cpp

```diff
@@ -180,59 +169,38 @@ namespace ImageProcessingAtom
                 m_image = new ImageToProcess(IImageObjectPtr(m_input->m_inputImage->Clone(mipsToClone)));
             }
 
-            break;
-        case StepGenerateIBL:
```

Comment:
You removed the case but didn't remove the enum value. Is that intentional?

Reply:
The StepGenerateIBL is not removed but moved to line 225. I want all the switch-cases follow the actually processing order in code.